api/cluster/container_test.go

```diff
@@ -27,12 +28,9 @@ import (
 	"k8s.io/client-go/kubernetes/fake"
 	fakecorev1 "k8s.io/client-go/kubernetes/typed/core/v1/fake"
 	ktesting "k8s.io/client-go/testing"
-
-	"github.com/caraml-dev/merlin/config"
```

Comment:
Oh I thought the placement of `"github.com/caraml-dev/merlin/config"` at the bottom was intentional - can we keep it that way? We usually segregate them like the following, right?

- Standard libraries
- OSS libraries
- "Local" libraries

Reply:
Same comment about the ordering of imports in other files in this PR as well.